.github/workflows/main.yml

```diff
@@ -60,9 +60,11 @@ jobs:
 
       - name: Install dependencies
         run: |
-          # HACK: this should be set already by the 'Use ocaml' step, but it is not
+          # HACK: the repo should be set already by the 'Use ocaml' step, but it is not
           # (this might be because of the cache)
+          opam repo remove default
```

Comment:
If the repo is added and removed it's already up-to-date. I recommend not adding this line, updating the url and updating the repo should be cheaper than cloning it whole.